src/puppetlabs/trapperkeeper/authorization/acl.clj

```diff
@@ -44,57 +46,58 @@
 
 ;; ACE creation
 
-(defn munge-name
-  [pattern]
-  (-> pattern (str/lower-case) (str/split #"\.") reverse vec))
+(schema/defn ^:private split-domain :- [String]
+  "Given a domain string, split it on '.' and reverse it. For examples,
+  'sylvia.plath.net' becomes ['net' 'plath' 'sylvia']. This is used for domain
+  matching."
+  [pattern :- String]
```

Comment:
An argument in favor of `pattern` is that it's storing "the thing to which another thing has to correspond" which seems like a proper use of the word. However, in our discipline, I agree that it almost always means "regex" which it only means here sometimes. I'm into `cynosure` but surely that is a smartass idea.